src/main/java/org/openrewrite/kotlin/internal/KotlinTreeParserVisitor.java

```diff
@@ -1300,48 +1300,35 @@ public J visitTypeAlias(KtTypeAlias typeAlias, ExecutionContext data) {
         List<J.Annotation> lastAnnotations = new ArrayList<>();
         Set<PsiElement> consumedSpaces = preConsumedInfix(typeAlias);
 
-        List<J.Modifier> modifiers = new ArrayList<>(mapModifiers(typeAlias.getModifierList(), leadingAnnotations, lastAnnotations, data));
+        List<J.Modifier> modifiers = mapModifiers(typeAlias.getModifierList(), leadingAnnotations, lastAnnotations, data);
         modifiers.add(new J.Modifier(randomId(), prefix(typeAlias.getTypeAliasKeyword(), consumedSpaces), markers, "typealias", J.Modifier.Type.LanguageExtension, emptyList()));
```

Comment:
If we don't need this modifier anymore, I think we should remove it. At the same time we should keep the code which is there to support type aliases created this way (for backwards compatibility). But we should also add a comment to these code blocks, so that nobody inadvertently removes them.

Reply:
I'm fine with removing the modifier -- it's mostly to preserve whitespace/comments. It seems odd to have either a `JLeftPadded<Name> name` or a `Space beforeTypeAlias` to support `internal typealias X = Y`, so I opted to keep the modifier in place.